README.md

```diff
@@ -5,8 +5,8 @@
 This repository contains the source of the FlowFuse website.
 
 It is built using [Tailwind CSS](https://tailwindcss.com/) and [Eleventy](https://www.11ty.dev/).
-It is hosted on GitHub Pages and every commit to the `main` branch is automatically
-deployed using GitHub Actions.
+
+It is hosted on Netlify with each commit to the `main` branch being automatically deployed to the live site.
```

Comment:
```suggestion
It is hosted on Netlify with each commit to the `main` branch being automatically deployed to the live site.

This works by a GitHub action automatically creating the `live` branch, which includes documentation pulled from the `maintenance` branch of the [flowforge/flowforge](https://github.com/flowforge/flowforge)
repository, when changes are pushed to `main`. 

Netlify is then configured to watch the `live` branch for any changes, once detected, it will automatically pull the contents of this branch 9docs included) and deploy to our production site.
```

Reply:
```suggestion
It is hosted on Netlify with each commit to the `main` branch being automatically deployed to the live site.

This works by a GitHub action automatically updating the `live` branch to includes documentation pulled from the `maintenance` branch of the [flowforge/flowforge](https://github.com/flowforge/flowforge)
repository, when changes are pushed to `main`.

Netlify is then configured to watch the `live` branch for any changes, once detected, it will automatically pull the contents of this branch (docs included) and deploy to our production site.
```